images/runtime/php/template.base.Dockerfile

```diff
@@ -27,7 +27,8 @@ RUN { \
 } >> /etc/apache2/apache2.conf
 
 # Install common PHP extensions
-RUN apt-get update \
+RUN apt-mark hold msodbcsql18 odbcinst1debian2 odbcinst unixodbc unixodbc-dev \
```

Comment:
What's the purpose of this `apt-mark hold` command? It may be worth adding a comment above this line to clarify why this is being done.

Reply:
This one is to mark specific packages as hold. This will prevent those packages from being upgraded when `apt-get upgrade` runs. I'll add comments to explain this. This will be reverted once the package source fixes the issue as well.